CHANGELOG.md

```diff
@@ -30,6 +35,8 @@ Thanks to ESLint, we’ve tidied up Metalsmith’s source code a bit.
 
 
 ### Security
+* Use `Buffer.from()` in supporting versions of Node.js possible (@Zearin)
```

Comment:
Why is this a security change?

Reply:
Well, according to the Node.js docs, there were security issues with using `new Buffer()`.  It had something to do with un-intuitive coercion of the input, depending on what was passed to `new Buffer`.

Come to think of it, since Metalsmith always reads from the filesystem, I don’t see how there was any risk to users.  My mindset at the time was simply “the old API was deprecated, so let’s follow the recommendation and replace `new Buffer` with `Buffer.from` whenever possible!”

Reply:
Yeah, I wasn't questioning the change itself, just its placement in the changelog. Obviously avoiding deprecated APIs whenever you can is always a good idea.

If there's some potential security risks related to `new Buffer` then I'm fine putting the change in this category, even if the overall risk to Metalsmith users is very minor.